tensorflow/contrib/io/python/kernel_tests/io_ops_test.py

```diff
@@ -0,0 +1,53 @@
+﻿# -*- coding: utf-8 -*-
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for tensorflow.contrib.io."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import zmq
```

Comment:
I think this might require a new dependency on our test infrastructure. @yifeif Do you know how to make this happen? (I'm assuming `pip install zmq` will do it.)